compiler/rustc_mir_build/src/lib.rs

```diff
@@ -11,7 +11,7 @@
 #![warn(unreachable_pub)]
 // tidy-alphabetical-end
 
-mod build;
+mod builder;
```

Comment:
Please add a comment here, e.g. "This used to be called `build`, but that causes GitHub's “Go to file” feature to silently ignore all files in this module, presumably because they are in a directory named `build`, which is mistaken for a build-output directory."

r=me with that change.